src/hooks/useHead.ts

```diff
@@ -100,9 +102,23 @@ export const useHead = ({ title, metas, language }: HeadObject) => {
   }, [memoizedMetas]);
 
   useEffect(() => {
+    if (amp) {
```

Comment:
Given this is repetition from the other location `useAMP`, would it make sense to export this and keep the logic centralized?